dev/devicelab/bin/tasks/build_ios_framework_module_test.dart

```diff
@@ -270,24 +270,36 @@ Future<void> _testBuildIosFramework(Directory projectDir, { bool isModule = fals
 
     await _checkBitcode(engineFrameworkPath, mode);
 
-    checkFileExists(path.join(
+    // TODO(jmagman): Remove ios-x86_64-simulator checks when the ARM simulator engine artifacts rolls.
+    final String x86SimulatorFramework = path.join(
       outputPath,
       mode,
       'Flutter.xcframework',
       'ios-x86_64-simulator',
       'Flutter.framework',
-      'Flutter',
-    ));
+    );
 
-    checkFileExists(path.join(
+    final String x86ArmSimulatorFramework = path.join(
       outputPath,
       mode,
       'Flutter.xcframework',
-      'ios-x86_64-simulator',
+      'ios-arm64_x86_64-simulator',
       'Flutter.framework',
-      'Headers',
-      'Flutter.h',
-    ));
+    );
+
+    final bool x86SimulatorBinaryExists = exists(File(path.join(x86SimulatorFramework, 'Flutter')));
```

Comment:
very minor nit: x86_64 should be shortened to x64, not x86. This looks good though.

Reply:
I'm going to delete this code within the next few days once we validate #85045 and are confident the roll won't be reverted.  Noted for future reference though.